Dockerfile

```diff
@@ -4,7 +4,7 @@
 #    docker build -f examples/$name/Dockerfile -t prometheus/golang-example-$name .
 
 # Builder image, where we build the example.
-FROM golang:1 AS builder
+FROM golang:1.12 AS builder
```

Comment:
I don't think it's a good idea to pin to golang 1 as breaking changes could happen once golang 1.13 is out.

Reply:
at the same time, client_golang should always work with the latest stable version of Go.

Reply:
And the dockerfile is only for the examples. I don't expect any problems with any Go 1.x version, given the Go stability guarantees.

Reply:
You could even turn the argument around: We deliberately want to build the examples with the latest Go version. If something doesn't work, that's a bug.

Reply:
Makes sense. I have fixed that.